Dockerfile

```diff
@@ -1,20 +1,44 @@
-FROM node:12-buster
+# Stage 1 - Create yarn install skeleton layer
+FROM node:14-buster AS packages
 
-WORKDIR /usr/src/app
+WORKDIR /app
+COPY package.json yarn.lock ./
 
-# Copy repo skeleton first, to avoid unnecessary docker cache invalidation.
-ADD yarn.lock package.json ./
-# Really annoying that you can't copy the tree easily using docker COPY command.
-# https://github.com/moby/moby/issues/15858#issuecomment-383690925
-ADD packages/backend/package.json ./packages/backend/package.json
-ADD packages/app/package.json ./packages/app/package.json
+# Uncomment this line if building a non create-app version
+# COPY packages packages
+COPY plugins plugins
 
-RUN yarn install --frozen-lockfile --production
+RUN find packages \! -name "package.json" -mindepth 2 -maxdepth 2 -print | xargs rm -rf
```

Comment:
I would appreciate a line or two as comment for this magic! 🙇 :)

Reply:
Will address this in the main repo when we do this

Reply:
This dockerfile will replace the main one for create-app etc

Reply:
Thanks!